### PR TITLE
Small fixes and linting

### PR DESCRIPTION
**Pull request by JasoonS**

### CODE REVIEW

File: contracts/contracts/NoLossDao_v0.sol [hunk 1]
```diff
@@ -39,7 +39,7 @@ contract NoLossDao_v0 is Initializable {
   mapping(address => address) public voteDelegations; // For vote proxy
 
   //////// Necessary to fund dev and miners //////////
-  address[] interestReceivers;
+  address[] interestReceivers; // in v0, the interestReceivers is the address of the miner.
```
Comment (moose-code) [inline contracts/contracts/NoLossDao_v0.sol:42]: Well, interestReceivers[0] is the miners address

File: contracts/contracts/NoLossDao_v0.sol [hunk 2]
```diff
@@ -156,7 +156,7 @@ contract NoLossDao_v0 is Initializable {
   modifier lockInFulfilled(address givenAddress) {
     require(
       iterationJoined[givenAddress] + 2 < proposalIteration,
-      'Benefactor only eligible to receive funds in later iteration'
+      'Benefactor has not fulfilled the minimum lockin period of 2 iterations'
```
Comment (moose-code) [inline contracts/contracts/NoLossDao_v0.sol:159]: This is 3 iterations

Comment (JasoonS) [reply contracts/contracts/NoLossDao_v0.sol:159]: Gotcha!

File: contracts/contracts/NoLossDao_v0.sol (hunk 1 above)
Comment (JasoonS) [reply contracts/contracts/NoLossDao_v0.sol:42]: That's what I meant 😬

**Pull request merged**